mmv1/third_party/terraform/services/container/resource_container_cluster.go.tmpl

```diff
@@ -4294,20 +4295,6 @@ func resourceContainerClusterUpdate(d *schema.ResourceData, meta interface{}) er
 		log.Printf("[INFO] GKE cluster %s fleet config has been updated", d.Id())
 	}
 
-	if d.HasChange("user_managed_keys_config") {
-		req := &container.UpdateClusterRequest{
-			Update: &container.ClusterUpdate{
-				UserManagedKeysConfig: expandUserManagedKeysConfig(d.Get("user_managed_keys_config")),
-			},
-		}
-		updateF := updateFunc(req, "updating GKE cluster user managed keys config.")
```

Comment:
Sorry, see my clarified comment on the reported issue.

It is possible to set these fields on an update.  For example, you can change the keys in the service_account_signing_keys list, to perform key rotations.  What you can't do is activate or deactivate the feature on an update (for right now).  So, you can't change from an empty service_account_signing_keys to a non-empty service_account_signing_keys (and vice-versa)

Reply:
Per discussion - the update behavior seems to be broken, so we'll need to stick with recreation for now.